src/foam/comics/v2/userFeedback/UserFeedback.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.comics.v2.userFeedback',
```

Comment:
From: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
"Naming Conventions
Package names are written in all lower case to avoid conflict with the names of classes or interfaces."

Reply:
Done.